CMakeLists.txt

```diff
@@ -9,14 +9,17 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 find_package(Boost REQUIRED)
+set(SSVM_DISABLE_AOT_RUNTIME ON)
+SET(SSVM_DISABLE_AOT_RUNTIME OFF CACHE BOOL "Build libfoo shared library")
```

Comment:
I have no idea about this line?
What does the `libfoo` mean?

Reply:
Wrong commit. Deleted.